db/schema.rb

```diff
@@ -18,22 +18,22 @@
   enable_extension "uuid-ossp"
 
   create_table "accounts", id: :uuid, default: "uuid_generate_v4()", force: :cascade do |t|
-    t.string   "name",                     default: "",                           null: false
-    t.string   "email",                    default: "",                           null: false
-    t.uuid     "uuid",                     default: "uuid_generate_v4()"
-    t.string   "encrypted_password",                                              null: false
+    t.string   "name",                                 default: "",                   null: false
+    t.string   "email",                                default: "",                   null: false
+    t.uuid     "uuid",                                 default: "uuid_generate_v4()"
+    t.string   "encrypted_password",                                                  null: false
     t.string   "reset_password_token"
     t.datetime "reset_password_sent_at"
     t.datetime "remember_created_at"
-    t.integer  "sign_in_count",            default: 0,                            null: false
+    t.integer  "sign_in_count",                        default: 0,                    null: false
```

Comment:
Avoid adding changes that are just whitespace changes to auto-generated files. Make sure you use `git add -p` when adding files so you can choose to *not* include changes that are not meaningful.